OpenTabletDriver.Configurations/Configurations/Huion/Kamvas Pro 19 (4k).json

```
@@ -0,0 +1,42 @@
+{
+    "Name": "Huion Tablet_GT1902",
```

Please follow the naming scheme for the rest of the Huion Kamvas tablets that we support.
```suggestion
    "Name": "Huion Kamvas Pro 19 (4K)",
```